src/app/pagination-item/pagination-item.component.html

```diff
@@ -0,0 +1,4 @@
+<li *ngIf=current class="pagination__current">
+  <a aria-current="true" (click)='onSelect(page)'>{{page}}</a>
```

Comment:
We should be able to conditionally set aria-content based on `current` using:

```suggestion
  <a [attr.aria-current]="current ? 'true' : null" (click)='onSelect(page)'>{{page}}</a>
```

according to https://stackoverflow.com/questions/36745734/how-to-add-conditional-attribute-in-angular-2